templates/github/.github/workflows/create-branch.yml.j2

```diff
@@ -66,3 +66,12 @@ jobs:
             Bump minor version
             [noissue]
           delete-branch: true
+
+      - name: add new backport label to the affected repository
+        run: |
+          curl -X POST \
+          -H "Accept: application/vnd.github.v3+json" \
+          -H "Authorization: Bearer {{ '${{ secrets.RELEASE_TOKEN }}' }}"\
```

Comment:
Is this the right token to use?
@dralley

Reply:
I believe so, we use this token to push to GitHub: https://github.com/pulp/plugin_template/blob/499255aa466ddf722376119f367d6ba32f34cba9/templates/github/.github/workflows/scripts/push_branch_and_tag_to_github.sh.j2#L10

Reply:
I had to test on my repo using ```secrets.GITHUB_TOKEN```, but I have left the RELEASE_TOKEN version here. I may be wrong, so feel free to correct me.